### PR TITLE
DEVDOCS-4719: [migrate] Tax V3 API & Tax Provider API, migrate syntax 

### PR DESCRIPTION
**Pull request by bc-andreadao**
# [DEVDOCS-4719]

Migration syntax for Tax endpoints (Tax V3 API, Tax Provider API)

[DEVDOCS-4719]: https://bigcommercecloud.atlassian.net/browse/DEVDOCS-4719?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by bc-tgomez — APPROVED** [review]

File: docs/api-docs/tax/tax-properties.mdx [hunk 1]
```diff
@@ -0,0 +1,403 @@
...
+
+### Create tax properties
+
+First, use the [Create tax properties](/api-reference/store-management/tax-properties/tax-properties/create-tax-properties) endpoint to add tax properties to the store. This requires the tax provider's `code` and a `display_name` that shoppers may see, depending on your theme, settings, and jurisdiction.  Optionally, you can add a description.
+
+The response provides an `id` for each tax property. Use the `id` to get, update, or delete a specific tax property.
+
+<Tabs items={[`Request`, `Response`]}>
+  <Tab>
+    ```http filename="Example request: Create tax properties" showLineNumbers
```
Comment (bc-traciporter) [inline docs/api-docs/tax/tax-properties.mdx:27]:
```suggestion
    ```http filename="Example request: Create tax properties" showLineNumbers copy
```

File: docs/api-docs/tax/tax-properties.mdx [hunk 2]
```diff
@@ -0,0 +1,403 @@
...
+      {
+        "code": "fuel-type",
+        "display_name": "Fuel Type",
+        "description": "Oil and Natural Gas Industry"
+      }  
+    ]
+    ```
+  </Tab>
+  <Tab>
+    ```json filename="Example response: Create tax properties" showLineNumbers
```
Comment (bc-traciporter) [inline docs/api-docs/tax/tax-properties.mdx:48]:
```suggestion
    ```json filename="Example response: Create tax properties" showLineNumbers copy
```

File: docs/api-docs/tax/tax-properties.mdx [hunk 3]
```diff
@@ -0,0 +1,403 @@
...
+  </Tab>
+</Tabs>
+
+### Update tax properties
+
+Send a request to the [Update tax properties](/api-reference/store-management/tax-properties/tax-properties/update-tax-properties) endpoint to modify a tax property's `code`, `display_name`, or `description`. The request updates only fields that you specify.   
+
+<Tabs items={[`Request`, `Response`]}>
+  <Tab>
+    ```http filename="Example request: Update tax properties" showLineNumbers
```
Comment (bc-traciporter) [inline docs/api-docs/tax/tax-properties.mdx:80]:
```suggestion
    ```http filename="Example request: Update tax properties" showLineNumbers copy
```

File: docs/api-docs/tax/tax-properties.mdx [hunk 4]
```diff
@@ -0,0 +1,403 @@
...
+  </Tab>
+</Tabs>
+
+### Get tax properties
+
+This endpoint supports batch operations. You can get all the tax properties in your store, or only specific tax properties. To get tax properties, send a request to the [Get tax properties](/api-reference/store-management/tax-properties/tax-properties/get-tax-properties) endpoint. To get only select tax properties, use the `id:in` query parameter.
+
+<Tabs items={[`Request`, `Response`]}>
+  <Tab>
+    ```http filename="Example request: Get tax properties" showLineNumbers
```
Comment (bc-traciporter) [inline docs/api-docs/tax/tax-properties.mdx:119]:
```suggestion
    ```http filename="Example request: Get tax properties" showLineNumbers copy
```

File: docs/api-docs/tax/tax-properties.mdx [hunk 5]
```diff
@@ -0,0 +1,403 @@
...
+  <Tab>
+    ```http filename="Example request: Get tax properties" showLineNumbers 
+    GET https://api.bigcommerce.com/stores/{{STORE_HASH}}/v3/tax/properties?id:in=1,2
+    X-Auth-Token: {{ACCESS_TOKEN}}
+    Content-Type: application/json
+    Accept: application/json
+    ```
+  </Tab>
+  <Tab>
+    ```json filename="Example response: Get tax properties" showLineNumbers
```
Comment (bc-traciporter) [inline docs/api-docs/tax/tax-properties.mdx:127]:
```suggestion
    ```json filename="Example response: Get tax properties" showLineNumbers copy
```

File: docs/api-docs/tax/tax-properties.mdx [hunk 6]
```diff
@@ -0,0 +1,403 @@
...
+  </Tab>
+</Tabs>
+
+### Delete tax properties
+
+To delete tax properties, send a request to the [Delete tax properties](/api-reference/store-management/tax-properties/tax-properties/delete-tax-properties) endpoint and use the `id:in` query parameter to specify the tax properties you want to delete.
+
+<Tabs items={[`Request`, `Response`]}>
+  <Tab>
+    ```http filename="Example request: Delete tax properties" showLineNumbers
```
Comment (bc-traciporter) [inline docs/api-docs/tax/tax-properties.mdx:170]:
```suggestion
    ```http filename="Example request: Delete tax properties" showLineNumbers copy
```

File: docs/api-docs/tax/tax-properties.mdx [hunk 7]
```diff
@@ -0,0 +1,403 @@
...
+  Tax properties are not stored on or retrievable with the product object.
+</Callout>
+
+### Update product with tax properties
+
+To attach tax properties to a product, send a request to the [Update product tax properties](/api-reference/store-management/tax-properties/product-tax-properties/update-product-tax-properties) endpoint. Use the same endpoint to modify a product's existing tax properties.
+
+<Tabs items={[`Request`, `Response`]}>
+  <Tab>
+    ```http filename="Example request: Update product with tax properties" showLineNumbers
```
Comment (bc-traciporter) [inline docs/api-docs/tax/tax-properties.mdx:201]:
```suggestion
    ```http filename="Example request: Update product with tax properties" showLineNumbers copy
```

File: docs/api-docs/tax/tax-properties.mdx [hunk 8]
```diff
@@ -0,0 +1,403 @@
...
+        "tax_properties": {
+          "alcohol-percentage": "10",
+          "netvolume-milliliters": "200"
+        }
+      }
+    ]
+    ```
+  </Tab>
+  <Tab>
+    ```json filename="Example response: Update product with tax properties" showLineNumbers
```
Comment (bc-traciporter) [inline docs/api-docs/tax/tax-properties.mdx:226]:
```suggestion
    ```json filename="Example response: Update product with tax properties" showLineNumbers copy
```

File: docs/api-docs/tax/tax-properties.mdx [hunk 9]
```diff
@@ -0,0 +1,403 @@
...
+  </Tab>
+</Tabs> 
+
+### Get product tax properties 
+
+To get the tax properties attached to a product, send a request to the [Get product tax properties](/api-reference/store-management/tax-properties/product-tax-properties/get-product-tax-properties) endpoint and use the `product_id:in` query parameter. 
+
+<Tabs items={[`Request`, `Response`]}>
+  <Tab>
+    ```http filename="Example request: Get product tax properties" showLineNumbers
```
Comment (bc-traciporter) [inline docs/api-docs/tax/tax-properties.mdx:267]:
```suggestion
    ```http filename="Example request: Get product tax properties" showLineNumbers copy
```

File: docs/api-docs/tax/tax-properties.mdx [hunk 10]
```diff
@@ -0,0 +1,403 @@
...
+  <Tab>
+    ```http filename="Example request: Get product tax properties" showLineNumbers 
+    GET https://api.bigcommerce.com/stores/{{STORE_HASH}}/v3/tax/products/properties?product_id:in=113,117
+    X-Auth-Token: {{ACCESS_TOKEN}}
+    Content-Type: application/json
+    Accept: application/json
+    ```
+  </Tab>
+  <Tab>
+    ```json filename="Example response: Get product tax properties" showLineNumbers
```
Comment (bc-traciporter) [inline docs/api-docs/tax/tax-properties.mdx:275]:
```suggestion
    ```json filename="Example response: Get product tax properties" showLineNumbers copy
```

File: docs/api-docs/tax/tax-properties.mdx [hunk 11]
```diff
@@ -0,0 +1,403 @@
...
+  #### Batch deletion
+  This endpoint removes **all** tax properties from a given product.
+</Callout>
+
+To remove tax properties from a product, send a request to the [Delete product tax properties](/api-reference/store-management/tax-properties/product-tax-properties/delete-product-tax-properties) endpoint and use the `product_id:in` query parameter. This disassociates all the tax properties from a product. 
+
+
+<Tabs items={[`Request`, `Response`]}>
+  <Tab>
+    ```http filename="Example request: Delete product tax properties" showLineNumbers
```
Comment (bc-traciporter) [inline docs/api-docs/tax/tax-properties.mdx:322]:
```suggestion
    ```http filename="Example request: Delete product tax properties" showLineNumbers copy
```

File: docs/api-docs/tax/tax-properties.mdx [hunk 12]
```diff
@@ -0,0 +1,403 @@
...
+  <Tab>
+    ```http filename="Example request: Delete product tax properties" showLineNumbers 
+    DELETE https://api.bigcommerce.com/stores/{{STORE_HASH}}/v3/tax/products/properties?product_id:in=117
+    X-Auth-Token: {{ACCESS_TOKEN}}
+    Content-Type: application/json
+    Accept: application/json
+    ```
+  </Tab>
+  <Tab>
+    ```http filename="Example response: Delete product tax properties" showLineNumbers
```
Comment (bc-traciporter) [inline docs/api-docs/tax/tax-properties.mdx:330]:
```suggestion
    ```http filename="Example response: Delete product tax properties" showLineNumbers copy
```

File: docs/api-docs/tax/tax-properties.mdx [hunk 13]
```diff
@@ -0,0 +1,403 @@
...
+  </Tab>
+</Tabs>
+
+## Tax Quotes
+
+BigCommerce sends product tax properties to request [tax estimates](/api-reference/providers/tax-provider-api/tax-provider/estimate) from a provider. The requests include tax properties for each item in a consignment. 
+
+The following request uses the Tax Provider API:
+
+```http filename="Example request: Get a tax estimate" showLineNumbers
```
Comment (bc-traciporter) [inline docs/api-docs/tax/tax-properties.mdx:342]:
```suggestion
```http filename="Example request: Get a tax estimate" showLineNumbers copy
```

File: docs/api-docs/tax/tax-rates-zones.mdx [hunk 14]
```diff
@@ -0,0 +1,462 @@
...
+
+### Create tax zones
+
+To add tax zones to a store, send a request to the [Create tax zones](/api-reference/store-management/tax-rates-and-zones/tax-zones/create-tax-zones) endpoint. The following example creates a subdivision-based zone.
+
+The response includes an `id` for each tax zone. Use the `id` to get, update, or delete a specific tax zone.
+
+<Tabs items={[`Request`, `Response`]}>
+  <Tab>
+    ```http filename="Example request: Create tax zones" showLineNumbers
```
Comment (bc-traciporter) [inline docs/api-docs/tax/tax-rates-zones.mdx:30]:
```suggestion
    ```http filename="Example request: Create tax zones" showLineNumbers copy
```

File: docs/api-docs/tax/tax-rates-zones.mdx [hunk 15]
```diff
@@ -0,0 +1,462 @@
...
+          "customer_groups": [
+            0
+          ]
+        }
+      }
+    ]
+    ```
+  </Tab>
+  <Tab>
+    ```json filename="Example response: Create tax zones" showLineNumbers
```
Comment (bc-traciporter) [inline docs/api-docs/tax/tax-rates-zones.mdx:71]:
```suggestion
    ```json filename="Example response: Create tax zones" showLineNumbers copy
```

File: docs/api-docs/tax/tax-rates-zones.mdx [hunk 16]
```diff
@@ -0,0 +1,462 @@
...
+  </Tab>
+</Tabs>
+
+### Update tax zones
+
+To modify a tax zone, send a request to the [Update tax zones](/api-reference/store-management/tax-rates-and-zones/tax-zones/update-tax-zones) endpoint. In the request body, specify the `id` of the zones you want to update. The request updates only the fields for which you supply values.
+
+<Tabs items={[`Request`, `Response`]}>
+  <Tab>
+    ```http filename="Example request: Update tax zones" showLineNumbers
```
Comment (bc-traciporter) [inline docs/api-docs/tax/tax-rates-zones.mdx:120]:
```suggestion
    ```http filename="Example request: Update tax zones" showLineNumbers copy
```

File: docs/api-docs/tax/tax-rates-zones.mdx [hunk 17]
```diff
@@ -0,0 +1,462 @@
...
+          "customer_groups": [
+            2
+          ]
+        }
+      }
+    ]
+    ```
+  </Tab>
+  <Tab>
+    ```json filename="Example response: Update tax zones" showLineNumbers
```
Comment (bc-traciporter) [inline docs/api-docs/tax/tax-rates-zones.mdx:145]:
```suggestion
    ```json filename="Example response: Update tax zones" showLineNumbers copy
```

File: docs/api-docs/tax/tax-rates-zones.mdx [hunk 18]
```diff
@@ -0,0 +1,462 @@
...
+  - All zones must cover a specified location; `locations` cannot be an empty array in the request body.
+</Callout>
+
+### Get tax zones
+
+[Get tax zones](/api-reference/store-management/tax-rates-and-zones/tax-zones/get-tax-zones) returns all tax zones by default. To return select tax zones, include the `id:in` query parameter.
+
+<Tabs items={[`Request`, `Response`]}>
+  <Tab>
+    ```http filename="Example request: Get tax zones" showLineNumbers
```
Comment (bc-traciporter) [inline docs/api-docs/tax/tax-rates-zones.mdx:200]:
```suggestion
    ```http filename="Example request: Get tax zones" showLineNumbers copy
```

File: docs/api-docs/tax/tax-rates-zones.mdx [hunk 19]
```diff
@@ -0,0 +1,462 @@
...
+    GET https://api.bigcommerce.com/stores/{{STORE_HASH}}/v3/tax/zones?id:in=2
+    X-Auth-Token: {{ACCESS_TOKEN}}
+    Accept: application/json
+    ```
+  </Tab>
+  <Tab>
+    ```json filename="Example response: Get tax zones" showLineNumbers
```
Comment (bc-traciporter) [inline docs/api-docs/tax/tax-rates-zones.mdx:207]:
```suggestion
    ```json filename="Example response: Get tax zones" showLineNumbers copy
```

File: docs/api-docs/tax/tax-rates-zones.mdx [hunk 20]
```diff
@@ -0,0 +1,462 @@
...
+  </Tab>
+</Tabs>
+
+### Delete tax zones
+
+To delete tax zones, send a request to the [Delete tax zones](/api-reference/store-management/tax-rates-and-zones/tax-zones/delete-tax-zones) endpoint. Use the `id:in` query parameter to specify the tax zones you want to delete. Deleting a tax zone cascades to remove all associated tax rates.
+
+<Tabs items={[`Request`, `Response`]}>
+  <Tab>
+    ```http filename="Example request: Delete tax zones" showLineNumbers
```
Comment (bc-traciporter) [inline docs/api-docs/tax/tax-rates-zones.mdx:256]:
```suggestion
    ```http filename="Example request: Delete tax zones" showLineNumbers copy
```

File: docs/api-docs/tax/tax-rates-zones.mdx [hunk 21]
```diff
@@ -0,0 +1,462 @@
...
+    DELETE https://api.bigcommerce.com/stores/{{STORE_HASH}}/v3/tax/zones?id:in=3,7
+    X-Auth-Token: {{ACCESS_TOKEN}}
+    Accept: application/json
+    ```
+  </Tab>
+  <Tab>
+    ```http filename="Example response: Delete tax zones" showLineNumbers
```
Comment (bc-traciporter) [inline docs/api-docs/tax/tax-rates-zones.mdx:263]:
```suggestion
    ```http filename="Example response: Delete tax zones" showLineNumbers copy
```

File: docs/api-docs/tax/tax-rates-zones.mdx [hunk 22]
```diff
@@ -0,0 +1,462 @@
...
+
+### Create tax rates
+
+To create tax rates, send a request to the [Create tax rates](/api-reference/store-management/tax-rates-and-zones/tax-rates/create-tax-rates) endpoint. Specify the rate name, rates of each tax class, and the associated tax zone. The `tax_zone_id` is the `id` received from calls to the [Get tax zones](/api-reference/store-management/tax-rates-and-zones/tax-zones/get-tax-zones) endpoint.
+
+The response includes an `id` for each tax rate. Use the tax rate `id` to get, update, or delete the associated tax rate.
+
+<Tabs items={[`Request`, `Response`]}>
+  <Tab>
+    ```http filename="Example request: Create tax rates" showLineNumbers
```
Comment (bc-traciporter) [inline docs/api-docs/tax/tax-rates-zones.mdx:281]:
```suggestion
    ```http filename="Example request: Create tax rates" showLineNumbers copy
```

File: docs/api-docs/tax/tax-rates-zones.mdx [hunk 23]
```diff
@@ -0,0 +1,462 @@
...
+        "enabled": true,
+        "name": "Sales Tax",
+        "priority": 1,
+        "tax_zone_id": 2
+      }
+    ]
+    ```
+  </Tab>
+  <Tab>
+    ```json filename="Example response: Create tax rates" showLineNumbers
```
Comment (bc-traciporter) [inline docs/api-docs/tax/tax-rates-zones.mdx:308]:
```suggestion
    ```json filename="Example response: Create tax rates" showLineNumbers copy
```

File: docs/api-docs/tax/tax-rates-zones.mdx [hunk 24]
```diff
@@ -0,0 +1,462 @@
...
+  </Tab>
+</Tabs>
+
+### Update tax rates
+
+To update tax rates, send a request to the [Update tax rates](/api-reference/store-management/tax-rates-and-zones/tax-rates/update-tax-rates) endpoint. In the request body, specify the `id` of the rates you want to update. The `tax_zone_id` is the `id` received from calls to the [Get tax zones](/api-reference/store-management/tax-rates-and-zones/tax-zones/get-tax-zones) endpoint.
+
+<Tabs items={[`Request`, `Response`]}>
+  <Tab>
+    ```http filename="Example request: Update tax rates" showLineNumbers
```
Comment (bc-traciporter) [inline docs/api-docs/tax/tax-rates-zones.mdx:340]:
```suggestion
    ```http filename="Example request: Update tax rates" showLineNumbers copy
```

File: docs/api-docs/tax/tax-rates-zones.mdx [hunk 25]
```diff
@@ -0,0 +1,462 @@
...
+        "id": 3,
+        "name": "Excise Tax",
+        "priority": 2,
+        "tax_zone_id": 4
+      }
+    ]
+    ```
+  </Tab>
+  <Tab>
+    ```json filename="Example response: Update tax rates" showLineNumbers
```
Comment (bc-traciporter) [inline docs/api-docs/tax/tax-rates-zones.mdx:364]:
```suggestion
    ```json filename="Example response: Update tax rates" showLineNumbers copy
```

File: docs/api-docs/tax/tax-rates-zones.mdx [hunk 26]
```diff
@@ -0,0 +1,462 @@
...
+  </Tab>
+</Tabs>
+
+### Get tax rates
+
+To get tax rates, send a request to the [Get tax rates](/api-reference/store-management/tax-rates-and-zones/tax-rates/get-tax-rates) endpoint. To get the tax rates for a subset of zones, pass a comma-separated string of the tax zone `id`s using the `tax_zone_id:in` query parameter.
+
+<Tabs items={[`Request`, `Response`]}>
+  <Tab>
+    ```http filename="Example request: Get tax rates" showLineNumbers
```
Comment (bc-traciporter) [inline docs/api-docs/tax/tax-rates-zones.mdx:396]:
```suggestion
    ```http filename="Example request: Get tax rates" showLineNumbers copy
```

File: docs/api-docs/tax/tax-rates-zones.mdx [hunk 27]
```diff
@@ -0,0 +1,462 @@
...
+    GET https://api.bigcommerce.com/stores/{{STORE_HASH}}/v3/tax/rates?tax_zone_id:in=2
+    X-Auth-Token: {{ACCESS_TOKEN}}
+    Accept: application/json
+    ```
+  </Tab>
+  <Tab>
+    ```json filename="Example response: Get tax rates" showLineNumbers
```
Comment (bc-traciporter) [inline docs/api-docs/tax/tax-rates-zones.mdx:403]:
```suggestion
    ```json filename="Example response: Get tax rates" showLineNumbers copy
```

File: docs/api-docs/tax/tax-rates-zones.mdx [hunk 28]
```diff
@@ -0,0 +1,462 @@
...
+  </Tab>
+</Tabs>
+
+### Delete tax rates
+
+To delete tax rates, send a request to the [Delete tax rates](/api-reference/store-management/tax-rates-and-zones/tax-rates/delete-tax-rates) endpoint. Use the `id:in` query parameter to specify the tax rates you want to delete.
+
+<Tabs items={[`Request`, `Response`]}>
+  <Tab>
+    ```http filename="Example request: Delete tax rates" showLineNumbers
```
Comment (bc-traciporter) [inline docs/api-docs/tax/tax-rates-zones.mdx:435]:
```suggestion
    ```http filename="Example request: Delete tax rates" showLineNumbers copy
```

File: docs/api-docs/tax/tax-rates-zones.mdx [hunk 29]
```diff
@@ -0,0 +1,462 @@
...
+    DELETE https://api.bigcommerce.com/stores/{{STORE_HASH}}/v3/tax/rates?id:in=3,5
+    X-Auth-Token: {{ACCESS_TOKEN}}
+    Accept: application/json
+    ```
+  </Tab>
+  <Tab>
+    ```http filename="Example response: Delete tax rates" showLineNumbers
```
Comment (bc-traciporter) [inline docs/api-docs/tax/tax-rates-zones.mdx:442]:
```suggestion
    ```http filename="Example response: Delete tax rates" showLineNumbers copy
```

File: docs/api-docs/tax/tax-settings.mdx [hunk 30]
```diff
@@ -8,49 +8,40 @@ This guide shows you how to use the Tax Settings API. For more info, see the [Ta
...
-```json title="Example response: Get tax settings" lineNumbers 
-{
-  "data": {
-    "tax_entered_with_prices": true,
-    "price_display_settings": {
-      "show_inclusive_in_control_panel": true,
-      "invoice_price_display_strategy": "ZONE"
+<Tabs items={[`Request`, `Response`]}>
+  <Tab>
+    ```http filename="Example request: Get tax settings" showLineNumbers
```
Comment (bc-traciporter) [inline docs/api-docs/tax/tax-settings.mdx:13]:
```suggestion
    ```http filename="Example request: Get tax settings" showLineNumbers copy
```

File: docs/api-docs/tax/tax-settings.mdx [hunk 31]
```diff
@@ -8,49 +8,40 @@ This guide shows you how to use the Tax Settings API. For more info, see the [Ta
...
+    GET https://api.bigcommerce.com/stores/{{STORE_HASH}}/v3/tax/settings
+    X-Auth-Token: {{ACCESS_TOKEN}}
+    Content-Type: application/json
+    Accept: application/json
+    ```
+  </Tab>
+  <Tab>
+  ```json filename="Example response: Get tax settings" showLineNumbers
```
Comment (bc-traciporter) [inline docs/api-docs/tax/tax-settings.mdx:21]:
```suggestion
  ```json filename="Example response: Get tax settings" showLineNumbers copy
```

File: docs/api-docs/tax/tax-settings.mdx [hunk 32]
```diff
@@ -8,49 +8,40 @@ This guide shows you how to use the Tax Settings API. For more info, see the [Ta
...
 
-<!--
-type: tab
-title: Request
--->
+<Tabs items={[`Request`, `Response`]}>
 
-```json title="Example request: Update tax settings" lineNumbers
+<Tab>
+```json filename="Example request: Update tax settings" showLineNumbers
```
Comment (bc-traciporter) [inline docs/api-docs/tax/tax-settings.mdx:44]:
```suggestion
```json filename="Example request: Update tax settings" showLineNumbers copy
```

File: docs/api-docs/tax/tax-settings.mdx [hunk 33]
```diff
@@ -63,13 +54,10 @@ Accept: application/json
...
+</Tab>
 
-<!--
-type: tab
-title: Response
--->
-
-```json title="Example response: Update tax settings" lineNumbers 
+<Tab>
+```json filename="Example response: Update tax settings" showLineNumbers
```
Comment (bc-traciporter) [inline docs/api-docs/tax/tax-settings.mdx:60]:
```suggestion
```json filename="Example response: Update tax settings" showLineNumbers copy
```

**Review by bc-traciporter — COMMENTED** [review]
left comments

**Review by slsriehl — APPROVED** [review]

**Pull request merged**